addons/mail/static/src/models/message/message.js

```diff
@@ -372,7 +372,7 @@ function factory(dependencies) {
          * @returns {string}
          */
         _computeDateFromNow() {
-            if (!this.date) {
+            if (!this.date || this.isTemporary) {
```

Comment:
The problem is that `date` is wrong, not the compute of date from now.

The date of the temporary message should be allowed to be empty, but the code in stable doesn't handle empty date correctly.

We already have done the proper change in master: https://github.com/odoo/odoo/pull/70069

But it might be hard to too dangerous to backport in stable?

Reply:
I was trying to do a minimum diff. I will try to back-port your change.